src/jdk.jpackage/share/native/common/app.cpp

```diff
@@ -23,6 +23,8 @@
  * questions.
  */
 
+#include "kludge_c++11.h"
```

Comment:
The name arose my curiosity, so I had to check out the file. Now that we indeed do have C++11 in the JDK (indeed, C++14), this should perhaps be revisited? (Not as part of this PR, of course)